Sources/Valet/SinglePromptSecureEnclaveValet.swift

```diff
@@ -138,6 +151,9 @@ public final class SinglePromptSecureEnclaveValet: NSObject {
     @objc(containsObjectForKey:)
     public func containsObject(forKey key: String) -> Bool {
```

Comment:
Might be worth noting in the header doc that it will return `false` if the keychain is not accessible.